snapshots/devmapper/metadata.go

```diff
@@ -50,9 +51,9 @@ var (
 
 var (
 	// ErrNotFound represents an error returned when object not found in meta store
-	ErrNotFound = errors.New("not found")
+	ErrNotFound = errdefs.ErrNotFound
 	// ErrAlreadyExists represents an error returned when object can't be duplicated in meta store
-	ErrAlreadyExists = errors.New("object already exists")
+	ErrAlreadyExists = errdefs.ErrAlreadyExists
 )
```

Comment:
Can we just remove these variables from the `snapshots/devmapper` pkg?

Reply:
The author did actually in the first iteration, to be safe I said maybe we can just assign them to the errdefs equivalents. I think it's probably fine to get rid of though, we don't have any callers here that check for these, was mostly thinking external, but not sure what someone would even be doing calling things from snapshotter/devmapper

Reply:
I've seen folks rely on some pretty strange implementation details, so I always err on the side of safety 🤣

Reply:
:yum::yum:
for now, containerd 1.x , keep compatible is good.
Maybe we can delete these dead code in 2.x?